pkg/manager/member/executor_manager.go

```diff
@@ -337,9 +349,11 @@ func (m *executorMemberManager) getNewExecutorStatefulSet(ctx context.Context, t
 
 	stsName := controller.TiflowExecutorMemberName(tcName)
 	stsLabels := label.New().Instance(instanceName).TiflowExecutor()
+
 	// can't directly use tc.Annotations here because it will affect tiflowcluster's annotations
 	// todo: use getStsAnnotations if we need to use advanced statefulset
-	stsAnnotations := map[string]string{}
+	//stsAnnotations := map[string]string{}
+	stsAnnotations := tc.Annotations
```

Comment:
It's a newly fixed bug here. Use `tc.Annotations` will affect tiflowcluster spec. It's not expected.
```suggestion
	stsAnnotations := map[string]string{}
```